In [21]:
import numpy as np
import pandas as pd

In [22]:
data=pd.read_csv("/content/Q9_b.csv")

In [23]:
data

Unnamed: 0          SP         WT
0            1  104.185353  28.762059
1            2  105.461264  30.466833
2            3  105.461264  30.193597
3            4  113.461264  30.632114
4            5  104.461264  29.889149
..         ...         ...        ...
76          77  169.598513  16.132947
77          78  150.576579  37.923113
78          79  151.598513  15.769625
79          80  167.944460  39.423099
80          81  139.840817  34.948615

[81 rows x 3 columns]

In [24]:
# Skewness
data.SP.skew()

1.6114501961773586

In [25]:
# Skewness
data.WT.skew()

-0.6147533255357768

In [26]:
# Skewness
data.skew()

Unnamed: 0    0.000000
SP            1.611450
WT           -0.614753
dtype: float64

## skewness value for speed and distance is 1.611450 and -0.614753 

In [27]:
# Kurtosis
data.SP.kurt()

2.9773289437871835

In [28]:
# Kurtosis
data.WT.kurt()

0.9502914910300326

In [29]:
# Kurtosis
data.kurt()

Unnamed: 0   -1.200000
SP            2.977329
WT            0.950291
dtype: float64


## Kurtosis value for speed and distance is 2.977329 and 0.950291
